packages/contracts-bedrock/contracts/deployment/BaseSystemDictator.sol

```diff
@@ -79,14 +79,14 @@ contract BaseSystemDictator is Ownable {
     /**
      * @notice Current step;
      */
-    uint256 public currentStep = 1;
+    uint8 public currentStep = 1;
```

Comment:
What does the `uint8` get us?

Reply:
Dumb reason, means the result from `ethers` isn't a BigNumber so I don't need to do `.toNumber()`. And we'll never have 255 steps here (dear lord I hope we never have 255 steps) so not a problem.